### PR TITLE
Style active links

### LINKED ISSUE #712: Redesign: Make the Current Page Active on the Navbar
This task involves making the link of the current page "active" when applicable. The style of an active link should just be the hover style for navbar links which is changing the color to black. Dont forget about the user dropdown when the user is logged in as well 😄 

---
### Working on this task

- [ ] If you want to work on this task, please claim it by commenting below   
- [ ] Branch off the redesign branch                                                                 
- [ ] Check out our [styleguide](https://staging-odin-project.herokuapp.com/styleguide)
- [ ] When you are submitting your pull request for this task please ensure the base branch you want to merge into is the redesign branch  

**Note**
* Please use the [contributing guide](https://github.com/TheOdinProject/theodinproject/wiki/Contributing-Guide) to get the app running on your local machine if you haven't already.

If you have any questions or need clarification about anything related to this task please contact the maintainers in our [contributing channel on Gitter](https://gitter.im/TheOdinProject/Contributing)

### PR DESCRIPTION
**Pull request by arun1595**
Fixes #712 

Tasks done: 
- Style active links with black color
- Extract navbar modal into a partial
- Make cosmetic changes to `head` partial

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def bootstrap_class_for(flash_type)
     bootstrap_classes.fetch(flash_type, custom_flash(flash_type))
   end
 
+  def active_class(path)
```
Comment (KevinMulhern) [inline app/helpers/application_helper.rb]: Can we include a unit test for this please?

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,77 +1,20 @@
...
-            <%= link_to 'Log In', login_path(ref: 'homenav'), class: 'nav-link' %>
-          </li>
-        <% end %>
-      </ul>
-    </div>
-  </div>
-</div>
-
-
+<%= render 'shared/navbar_modal' %>
```
Comment (KevinMulhern) [inline app/views/shared/_navbar.html.erb]: Nice Refactor 👍

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (arun1595) [reply app/helpers/application_helper.rb]: Sure :)

**Comment by arun1595** [comment]
@KevinMulhern PTAL :smile:

**Comment by KevinMulhern** [comment]
👍 LGTM
Thanks @arun1595

**Pull request merged**